dtonator/src/main/java/com/bizo/dtonator/GenerateDto.java

```diff
@@ -126,8 +132,13 @@ private void addCopyOfMethod() {
       if (hasMoreThanOneType) {
         m.body.line("if (o instanceof {}) {", c.getDtoType());
       }
-      // first make copies of children if needed
-      for (final DtoProperty dp : c.getAllProperties()) {
+
+      List<DtoProperty> properties = new ArrayList<DtoProperty>();
```

Comment:
Why this change?

Reply:
This looks like a leftover change from something else I tried out and discarded. Go ahead and roll this back if you want.